<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveStyleForm</class>
 <widget class="QDialog" name="SaveStyleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>269</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save Style</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Style Number</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_styleNo"/>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_4"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_styleID"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_2"/>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_1"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_2"/>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pushButton_submitToServer">
     <property name="text">
      <string>Submit to server</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_4"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Region 3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Region</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Region 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_1"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Region 4</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Region 2</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEdit_Tension_3"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_RPM_3"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Tension</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_size"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>RPM</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Style ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_material"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Fold width</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_FoldWidth"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>RGB</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_RGB"/>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="txt_debug"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
